q2_types/feature_table/_transformer.py

```diff
@@ -42,8 +52,31 @@ def _2(ff: BIOMV1Format) -> biom.Table:
 # appropriate).
 @plugin.register_transformer
 def _3(ff: BIOMV1Format) -> pd.DataFrame:
-    table = _parse_biom_table(ff)
+    table = _parse_biom_table_v100(ff)
+    return _table_to_dataframe(table)
+
+
+def _table_to_dataframe(table: biom.Table) -> pd.DataFrame:
```

Comment:
Can this be moved up with the other utility functions? I know it sounds nitpicky but we want to be able to use this plugin as a guide/standard for other plugin development efforts.